docarray/array/storage/elastic/backend.py

```diff
@@ -166,22 +172,15 @@ def _build_schema_from_elastic_config(self, elastic_config):
             ] = index_options
         return da_schema
 
-    def _build_client(self):
```

Comment:
I think we can keep this method and do `self._client = self._build_client()`, while still keeping `_build_index()`? That way we don't need to repeat these lines down in line 289